DatadogObjc/Sources/RUM/RUM+objc.swift

```diff
@@ -378,7 +378,7 @@ public class DDRUMConfiguration: NSObject {
     }
 
     @objc public var appHangThreshold: TimeInterval {
-        set { swiftConfig.appHangThreshold = newValue }
+        set { swiftConfig.appHangThreshold = newValue == 0 ? nil : newValue }
```

Comment:
I'm not sure this is related, but I see a `minAppHangThreshold` set to `0.1` in `RUMInstrumentation`. Can it be bypassed and actually set to `0`?

Reply:
this is per documentation https://github.com/DataDog/dd-sdk-ios/blob/085da2c4a390babc3bcc6097fc9f6f3a34d94a8e/DatadogRUM/Sources/RUMConfiguration.swift#L124-L125 which says to use `nil` value to disable it.

Reply:
Got it, thanks!